kahuna/public/js/services/api/collections-api.js

```diff
@@ -134,8 +137,8 @@ collectionsApi.factory('collections',
         addChildTo,
         isDeletable,
         removeFromList,
-        addImageIdsToCollection,
-        addImagesToCollection,
+        addCollectionUsingImageIds,
+        addCollectionUsingImageResources,
```

Comment:
Should it be add *to* collection? Else I wrongly read it as adding (creating) a new collection.